src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/PropertyManager.java

```diff
@@ -171,6 +171,11 @@ public Object getProperty(String property){
         /** Check to see if the property is managed by the security manager **/
         String propertyValue = (fSecurityManager != null) ?
                 fSecurityManager.getLimitAsString(property) : null;
+        /** Check to see if the property is managed by the security property manager **/
+        if (propertyValue == null){
```

Comment:
Let's add space before `{`

Reply:
Added. Also hit the format button for the class, that stirred up a hornet's nest, ended up making more (unnecessary) changes. Fortunately, it's confined to format and comments (not affecting the functionality).

Reply:
Thanks, I didn't see the split in the commits.
IMHO for an existing class, its a bad idea to ask an IDE to reformat a whole file; just too many gratuitous changes.

Reply:
Very true. When you look at these legacy code, it's quite tempting as you make style changes, e.g. adding a space next to more such things in the adjacent code. But yeh, must resist the urge :-)

Thanks again!